testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/harness.js

```diff
@@ -57,6 +57,19 @@ NX.TestHarness.start(
     },
 
     {
+      group: 'Search',
+      items: [
+        {
+          group: 'FT',
+          items: [
+            { url: 'testsuite-oss/search/search_basic.t.js' }
```

Comment:
Damn :( 
We have been hopeful that we could keep both paths (mvn and browser) open to having all tests pass, which unfortunately is difficult without the ability to upload in the UI